roles/app_deployment/tasks/vms.yml

```diff
@@ -0,0 +1,30 @@
+- name: Initialize image
+  set_fact:
+    dvs:
+      - name: "ansible-dv-{{ item.name }}"
+        source:
+          http:
+            url: "{{ item.image.url }}"
+        pvc:
+          accessModes:
+            - ReadWriteOnce
+          storage: "{{ item.image.storage }}"
+  when: item.image is defined
+
+- name: Create virtual machines
+  kubevirt_vm:
+    state: "{{ item.state | default('running') }}"
+    wait: "{{ item.wait | default(true) }}"
+    name: "{{ item.name | mandatory }}"
+    namespace: "{{ item.namespace | default('default') }}"
+    memory: "{{ item.memory | default(omit) }}"
```

Comment:
What about cpu cores?

Reply:
This is basic layout WIP. I will be adding other stuff, like cloud init, affinity, network etc.

Reply:
My idea is to merge this and add each feature in seprate PR so you can participate on small code reviews.